lib/service/html.php

```diff
@@ -30,7 +30,7 @@ class Html {
 	 */
 	public function convertLinks($data) {
 		$linker = new UrlLinker(true, false);
-		$data = $linker->linkUrlsAndEscapeHtml($data);
+		$data = $linker->linkUrlsInTrustedHtml($data);
```

Comment:
@DeepDiver1975 This will **not** escape HTML found in `$data`, but you use HTMLPurifier so it should be safe (as long as purifier is configured properly).